Update serverless installation docs (#8116)
DataDog/documentation: content/en/serverless/installation/nodejs.md

      text: 'Installing Ruby Serverless Monitoring'
---

> After you have [installed the AWS integration][1], choose one of the following method to instrument your application to send metrics, logs, and traces to Datadog.

Reviewer: ```suggestion
After you have [installed the AWS integration][1], choose one of the following methods to instrument your application to send metrics, logs, and traces to Datadog.
```

yarn add datadog-lambda-js
```

See the [latest release][3].

### Configure the Function

1. Set your function's handler to `/opt/nodejs/node_modules/datadog-lambda-js/handler.handler` if using the layer, or `node_modules/datadog-lambda-js/dist/handler.handler` if using the package.

> 1. Set the environment variable `DD_LAMBDA_HANDLER` to your original handler, e.g., `myfunc.handler`.

Reviewer: ```suggestion
2. Set the environment variable `DD_LAMBDA_HANDLER` to your original handler, for example, `myfunc.handler`.
```

> 1. Set environment variable `DD_TRACE_ENABLED` to `true`.

Reviewer: ```suggestion
3. Set the environment variable `DD_TRACE_ENABLED` to `true`.
```

> 1. Set environment variable `DD_FLUSH_TO_LOG` to `true`.

Reviewer: ```suggestion
4. Set the environment variable `DD_FLUSH_TO_LOG` to `true`.
```

5. Optionally add a `service` and `env` tag with appropriate values to your function.

### Subscribe the Datadog Forwarder to the Log Groups

You need to subscribe the Datadog Forwarder Lambda function to each of your function’s log groups, in order to send metrics, traces and logs to Datadog.

1. [Install the Datadog Forwarder if you haven't][4].
2. [Ensure the option DdFetchLambdaTags is enabled][5].
3. [Subscribe the Datadog Forwarder to your function's log groups][6].

[1]: https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html
[2]: https://github.com/DataDog/datadog-lambda-layer-js/releases
[3]: https://www.npmjs.com/package/datadog-lambda-js
[4]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder
[5]: https://docs.datadoghq.com/serverless/troubleshooting/installing_the_forwarder/#ddfetchlambdatags
[6]: https://docs.datadoghq.com/integrations/amazon_web_services/?tab=automaticcloudformation#send-aws-service-logs-to-datadog
{{% /tab %}}
{{< /tabs >}}

> ## Quick Start

Reviewer 2: ```suggestion
## Explore Datadog Serverless Monitoring
```

> ## Quick Start

Reviewer: ```suggestion
## Viewing your data
```

Reviewer 2: Let's keep it as quick start for now, but we should revisit this section and chat about what we want it to look like longer term @apigirl.

Reviewer 2: For now, let's call this "Explore Datadog Serverless Monitoring" in line with what APM and other sections are doing.

> After you have configured your function following the steps above, you can view the enhanced metrics, logs and traces on the [Serverless page][2]. If you would like to submit a custom metric or manually instrument a function, see the sample code below.

Reviewer: ```suggestion
After you have configured your function, you can view the enhanced metrics, logs and traces on the [Serverless page][2]. If you would like to submit a custom metric or manually instrument a function, see the sample code below.
```

> After you have configured your function following the steps above, you can view metrics, logs and traces on the [Serverless page][2]. If you would like to submit a custom metric or manually instrument a function, see the sample code below.

Reviewer 2: ```suggestion
After you have configured your function following the steps above, you can view metrics, logs and traces on the [Serverless page][2]. If you would like to submit a custom metric or manually instrument a function, see the sample code below:
```
